requests-responses.md

```diff
@@ -38,6 +38,7 @@ App’s request message is **InitialRequest**.
 ```tsx
 type ConnectRequest = {
   manifestUrl: string;
+  returnStrategy: 'default' | 'disabled' | '<my-return-url>';
```

Comment:
```suggestion
  return: 'default' | 'disabled' | '<my-return-url>';
```

Reply:
Probably 'alwaysBack' will be clearer than 'back', what do you think?

Reply:
What are the exact semantics of this? “Return where the system called from”? Then something like “system-back” would be more precise.

Not sure “always” has relevance here — this option is meant for a single interaction, it is not “sticky”.